games-rpg/flare/additional-files/flare.rdef.in

```diff
@@ -9,10 +9,10 @@ resource app_version {
 	internal = 0,
 
 	short_info = "Flare",
-	long_info = "@LONG_INFO@"
+	long_info = "Free/Libre Action Roleplaying Engine"
```

Comment:
Changes in this file are not needed, those are done with sed magic in the recipe, if you feel the SUMMARY is wrong, adjust that in the recipe.

Reply:
Is sed even needed if the string will always be the same either way? I don't see a point in using sed in cases like this

Reply:
The point is to reuse the value of `$SUMMARY` from the recipe and not have the same string in two places.

Reply:
I'm sorry, that makes no sense to me, you end up with the same string in two places either way. I guess it would be useful in case the project ever changes its name, but even if that was the case there's no guarantee one could reuse the summary in the same way, ie. maybe thy rename themselves to just "FLARE" and then it would be a bad string for "long_info" either way.

Reply:
SUMMARY has to "always" be at least 3 words (guidelines), these sed things and rdef's have been in use to ease the updates on future versions, nothing wrong with that, if upstream changes it's name or the like, the recipe should also reflect that, it's like "libVersion" you could drop that, but that would mean on a next update you would have to change all instances where libVersion is called from/declared in.

Reply:
To me it makes sense for version numbers and strings that are meant to change from version to version, long info and summary would stay the same regardless of recipe updates and if they change there's no actual guarantee the sed thing would still apply, imo it is just an extra command in the recipe to change a string that is very unlikely to ever change.

Reply:
I fail to see why the sed thing would fail to apply?

Reply:
I fail to see why we need an extra command to change a string that's staying the same regardless

Reply:
I get that, I'm not willing to back down either. :)
Sorry guys, I don't see any actual benefit of using variables for fields like that, simple as.

Reply:
Just imagine the rdef.in is a template you can't edit, because it's the same for every recipe. We might actually drop the rdef.in in favor of recipe scriptlet in the future.

Reply:
It's the same for every recipe you say... to me that's a confusing statement, this is the first rdef.in I've seen with so many variables in it, most only have version numbers as variables and that's it. Version numbers come and go, sure, but is there any point in having a variable that should never change? That's my issue with it, fields like long_info and app_signature realistically have little reason to change overtime, unless a project shifts focus entirely.

Reply:
So will you update every rdef.in in the repo because they are confusing to you?

Reply:
You're saying every rdef.in in this repo has app_signature as a variable then?

Reply:
Just for reference: there is a counter-example that I wrote: https://github.com/haikuports/haikuports/commit/b3510feb4985b2791f8f31dbfcf3950025213fdd

The reason there is that perl module versions (which is used as `$portVersion`) are weird and aren't equal to the actual version of the app (perl module versions are single floating point numbers, not usual version numbers). So in the end I didn't even bother with `.rdef.in` files and instead just used `.rdef` files and copied them directly. But, admittedly, that is a special case-